src/domain/portfolio/entities/portfolio.entity.ts

```diff
@@ -0,0 +1,5 @@
+import { z } from 'zod';
+
+export const PortfolioSchema = z.unknown();
```

Comment:
This will be populated after scaffolding the UI as we may not use every field from the API.